scripts/fetch_and_build_deb

```diff
@@ -49,16 +49,20 @@ fi
 # populate variables from packaging metadata file
 # shellcheck source=/dev/null
 source /buildfiles/pkgvars
-
+source /buildfiles/supported-postgres.txt
 # set default values for certain packaging variables
 declare pkglatest # to make shellcheck happy
 pkgname="${deb_pkgname:-${pkgname}}"
 hubproj="${hubproj:-${pkgname}}"
 nightlyref="${nightlyref:-master}"
-releasepg="${releasepg:-11,12,13}"
-nightlypg="${nightlypg:-${releasepg}}"
+releasepg="${release_versions:-${releasepg}}"
```

Comment:
I think your intention is not so easy to read here. You need to explain this better.

Do you want the `release_versions` from to overwrite the value supplied in `releasepg`? Can you do that in a cleaner fashion?

Reply:
My intention is to use pkgvars releasepg values if file not found

Reply:
Changed variable names and added some comments

Reply:
After addressing all comments, I will manually create packaging-test images to test the developments made here on https://github.com/citusdata/tools/pull/167